src/plugins/dashboard/public/dashboard_actions/filters_notification_popover_contents.tsx

```diff
@@ -53,6 +57,7 @@ export function FiltersNotificationPopoverContents({ context }: FiltersNotificat
           const language = getAggregateQueryMode(embeddableQuery);
           setQueryLanguage(language);
           setQueryString(embeddableQuery[language as keyof AggregateQuery]);
+          setDisableEditButton(true);
```

Comment:
It looks like we'll only end up here if the language is `SQL` or `ESQL`, is that correct?

Reply:
Exactly Devon, the first condition is about KQL, the rest is text based languages!